client/src/components/feedback/send.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from 'react';
+import { Button, Form } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Api from '../../services/api';
+import '../../styles/feedback-send.css';
+import MyBadge from '../shared/my-batch';
+import CancelButton from '../shared/button/cancel';
+
+class FeedbackSend extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      description: '',
+      isLoading: false,
+      timerCount: 3,
+      showTimer: true,
+      tooShortFeedback: false,
+      tooLongFeedback: false,
+      feedbackSent: false,
+      feedbackNotSent: false,
```

Comment:
I think you can make here one variable isSent.

Reply:
Unfortunately, I can't. This is about absense of `null` value of our `bool`. These twoo `bools` are need to show or not to show two different 'Badges' in order to components state. Initial state of only one bool should be false as we have nothing to show to user yet (while attemt to send feedback is made).

It is possible to connect this with 'attemtCount' bool, but logic will be more complicated.